client/src/Components/App/App.js

```diff
@@ -49,10 +49,14 @@ const App = () => {
   }, [feeling]);
 
   useEffect(() => {
-    getAllEntries()
-    .then(data => setUserLogs(data));
+    updateLogs();
   }, []);
 
+  const updateLogs = () => {
```

Comment:
Love this function, very semantic/clear what it's purpose is. Thank you for keeping our data model in tact.